actions/apt_action.go

```diff
@@ -49,6 +56,49 @@ func (apt *AptAction) Run(context *debos.DebosContext) error {
 	aptOptions = append(aptOptions, "install")
 	aptOptions = append(aptOptions, apt.Packages...)
 
+	if apt.LocalPackagesDir != "" {
+		localpackagesdir := path.Join(context.RecipeDir, apt.LocalPackagesDir)
+		destination, err := debos.RestrictedPath(context.Rootdir, "/usr/local/debs")
+		if err != nil {
+			return err
+		}
+		defer os.RemoveAll(destination)
+
+		err = debos.CopyTree(localpackagesdir, destination)
+		if err != nil {
+			return err
+		}
+
+		currentDir, _ := os.Getwd()
+		os.Chdir(destination)
+
+		// apt-ftparchive tries to read /etc/apt/apt.conf.d, add this path temporarily to prevent warnings
+		os.MkdirAll("/etc/apt/apt.conf.d", 0755)
+		defer os.RemoveAll("/etc/apt/apt.conf.d/")
```

Comment:
if this is only a warning i wonder if this should be a separate patch since debootstrap action also moans about this
also, this is in the middle of a `chdir`, for clarity move it above